src/modules/tokens/models/esdtToken.model.ts

```diff
@@ -23,6 +23,10 @@ export class EsdtToken implements IEsdtToken {
     derivedEGLD: string;
     price?: string;
     previous24hPrice?: string;
+    previous7dPrice?: string;
+    priceChange24h?: string;
+    previous24hVolume?: string;
```

Comment:
this field should be `volumeUSD24h` which means the token's volume in the past 24 hours

Reply:
updated field name and the compute method (sum of last 24h values)